salt/utils/aws.py

```diff
@@ -45,6 +45,7 @@
     'InsufficientAddressCapacity',
     'InsufficientReservedInstanceCapacity',
 ]
+AWS_METADATA_TIMEOUT = 3.05
```

Comment:
@notpeter, thanks for the fix.  Is it possible to add this to the `__opts__` data rather than create a global file variable?